packages/shared/lib/utils/encryption.manager.ts

```diff
@@ -20,14 +21,15 @@ class EncryptionManager extends Encryption {
         return Boolean(this?.key && this.key.length > 0);
     }
 
-    public encryptEnvironment(environment: Environment) {
+    public async encryptEnvironment(environment: Environment) {
         if (!this.shouldEncrypt()) {
             return environment;
```

Comment:
when no encryption bc of this condition we are now not filling `secret_key_hashed`

Reply:
you are right. I must leave it there https://github.com/NangoHQ/nango/pull/2119/files#diff-f3d9e285186619d6dfde44bdbbb720119863b18a3a80a41025f4a9c83e13705aL240

Reply:
did you merge without changing?

Reply:
I am an idiot. I am made the change and didn't push it. 🤦‍♂️